src/Mailgun/Model/Stats/TotalResponseItem.php

```diff
@@ -43,9 +43,9 @@ public static function create(array $data)
     {
         return new self(
             isset($data['time']) ? new \DateTime($data['time']) : null,
-            isset($data['accepted']) ? $data['accepted'] : null,
-            isset($data['delivered']) ? $data['delivered'] : null,
-            isset($data['failed']) ? $data['failed'] : null
+            isset($data['accepted']) ? $data['accepted'] : [],
+            isset($data['delivered']) ? $data['delivered'] : [],
+            isset($data['failed']) ? $data['failed'] : []
```

Comment:
Question: why to set `[]` instead of `null`? IMO if we don't have that array key, then `null` is fine.

Reply:
The `TotalResponseItem` class requires array parameters in its constructor.
Without this there were errors as null was not accepted as a suitable constructor parameter.

This PR fixes two bugs ;)

Reply:
That makes sense, however it means one more thing:

What about another update? Instead of:

```php
private function __construct(
    \DateTime $time,
    array $accepted,
    array $delivered,
    array $failed
)
```

the Constructor could be this:

```php
private function __construct(
    \DateTime $time,
    array $accepted = null,
    array $delivered = null,
    array $failed = null
)
```

So in this way we can think about the option that a given array key could not be passed.

Reply:
In fact, this returns `null` if the keys are not provided:
https://github.com/mailgun/mailgun-php/blob/master/src/Mailgun/Model/Stats/TotalResponseItem.php#L42-L50

Reply:
Allowing null values will require users to check if return of `TotalResponseItem::getAccepted()`, `TotalResponseItem::getDelivered()`, `..` is null before  iterating on stats items.

Reply:
Good point, but before to go ahead with this, I would say to confirm - based on the official API docs - if this scenario could happen, because if not then we won't be providing a "pure" SDK that should provide the same behaviour that we could get running HTTP Requests to the API.